Cargo.toml

```diff
@@ -34,6 +34,8 @@ diesel2 = { default-features = false, optional = true, package = "diesel", versi
 ndarray = { default-features = false, optional = true, version = "0.15.6" }
 num-traits = { default-features = false, features = ["i128"], version = "0.2" }
 postgres = { default-features = false, optional = true, version = "0.19" }
+proptest = { default-features = false, optional = true, features = ["std"], version = "1.0" }
+proptest-derive = { default-features = false, optional = true, version = "0.3" }
```

Comment:
Do you need proptest-derive? I don't believe it's used in this implementation.

Reply:
Just reading about that library, I'm pretty sure that it's probably unneeded and can be removed! :) It looks like you've implemented `Arbitrary` by hand so need for this too.

Reply:
```suggestion
```

Reply:
No I don't. I originally was gonna use it but wrote the implementation by hand and didn't remove the depend.